packages/keycodes/README.md

```diff
@@ -0,0 +1,13 @@
+# @wordpress/keycodes
+
+Keycodes utilities for WordPress.
```

Comment:
Can we say *what* it's used for, in case someone reading this isn't familiar with what "key codes" means? I have some thoughts… I'll just add them in a commit 😄